lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -90,52 +93,50 @@ defmodule ExUnit.Runner do
   # Loop expecting messages from the spawned modules. Whenever
   # a module has finished executing, decrease the taken modules
   # counter and attempt to spawn new ones.
-  defp wait_until_available(config, taken) do
+  defp wait_until_available(modules, taken, config) do
     receive do
       {_pid, :module_finished, _test_case} ->
-        loop(config, taken - 1)
+        loop(modules, taken - 1, config)
     end
   end
 
-  defp spawn_modules(config, modules, taken) do
-    pid = self()
-
+  defp spawn_modules(modules, modules_left, taken, config) do
```

Comment:
`remaining_modules` or `modules_remaining` may be a better name then `modules_left`.